packages/host/app/components/operator-mode/code-submode.gts

```diff
@@ -829,6 +870,15 @@ export default class CodeSubmode extends Component<Signature> {
       .empty-container > :deep(svg) {
         --icon-color: var(--boxel-highlight);
       }
+      :global(.accordion-item-content) {
```

Comment:
I think this is just moving from one place to another but a global style can be avoided:
```suggestion
      .accordion-item :deep(.accordion-item-content) {
```